include/bluetooth/mesh/mesh_time.h

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
This header should be renamed to `time.h`

Reply:
I believe this was tried, but failed, as it conflicted with stdlib's time.h. Could be a fluke though.

Reply:
If the include is "bluetooth/mesh/time.h"?

Reply:
Lets give it a try